main.go

```diff
@@ -124,6 +124,12 @@ func init() {
 			DefValue:    24 * time.Hour,
 			Description: "The timeout before discarding deal with no progress",
 		},
+		{
+			Name:     "est-download-speed",
+			DefValue: "5MB",
```

Comment:
I'd be more conservative here. The existing default was 3hs.
The usual CAR files are ~32GiB, and usually, the same miner receives a batch of things to download.. so more than one thing downloading at the same time. Maybe 1MB/s as default.

I believe we have much more room for improvement in all this download+import... like, a miner downloading 10 CAR files at the same time not only is a risk for download, but I'd imagine importing 10 CAR files later at the same time might nuke their lotus node.

We could always cap the amount of things done concurrently as I suggested some times.. but that's another feature that requires a bit more work.

Reply:
So for 1MB/s would be 9hs. If is downloading 3 CAR files at the same time, is an avg of 3hs per CAR. (all hand-wavy...)
Maybe too conservative... but around there, could be a bit more.

Reply:
1MB sounds good. it is what I originally set. good point on concurrent downloads. Maybe just limit `MaxDataURIFetchConcurrency` to 3? 10 concurrent downloads won't make sense if the bottleneck is bandwidth.

Reply:
Yep, I believe we need that flag.

Reply:
At some point might be good to stop bidding if you already have X things waiting for download [just sharing thought for a next feature]

Reply:
It's an existing constant we can simply change right now.

Reply:
Oh, right didn't recall. LGTM to switch to 3.